sds_in_a_box/sds_in_a_box_stack.py

```diff
@@ -115,6 +115,31 @@ def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
                                                       )
                                         )
         indexer_lambda.apply_removal_policy(cdk.RemovalPolicy.DESTROY)
+        
+        # Adding a lambda for uploading files to the SDS
+        upload_api_lambda = lambda_alpha_.PythonFunction(self,
+                                      id="APILambda",
```

Comment:
Can we name it to `UploadAPILambda`? I was trying to test upload lambda by cdk deploy and looking at the resources created by it. It took me a while to find upload api lambda function.

Reply:
Whoops, yeah good catch